include/oneapi/dpl/internal/distributed_ranges_impl/shp.hpp

```diff
@@ -0,0 +1,17 @@
+// SPDX-FileCopyrightText: Intel Corporation
+//
+// SPDX-License-Identifier: BSD-3-Clause
+
+#pragma once
+
+// #include "detail/logger.hpp"
```

Comment:
I guess this can be deleted since the file doesn't exist.

Reply:
ok, applied